docs/hermes

```diff
@@ -0,0 +1 @@
+.
```

Comment:
?

Reply:
This is Github's poor way of displaying a symlink.  The plan is to host the docs on Github Pages, where it will live at `https://articulate.github.io/hermes`.  Notice the `/hermes` base path.  Including this symlink allows the docs to be displayed correctly both on Github Pages with a base path and on `localhost` where it lives at the root.

At least... I hope so.  We haven't deployed it to Github Pages yet.  😅

Reply:
Oh... do we care about OS's that don't have symlinks? Or do the docs have to be docker-ized?

Reply:
I haven't been running the docs in Docker.  According to the Google, Windows now supports symlinks.  Are there others that don't?

Reply:
Whaaaat... I'll be quiet now.